src/app/contexts/RequestContext/index.test.jsx

```diff
@@ -1,102 +1,110 @@
+import * as getOriginContext from './getOriginContext';
+import * as getEnv from './getEnv';
 
 const { RequestContextProvider, RequestContext } = require('./index');
 
-const renderWithContextProvider = (
-  node,
-  {
-    platformString,
-    bbcOrigin,
-    pageType,
-    statsDestination,
-    statsPageIdentifier,
-  },
-) => (
-  <RequestContextProvider
-    platform={platformString}
-    bbcOrigin={bbcOrigin}
-    pageType={pageType}
-    statsDestination={statsDestination}
-    statsPageIdentifier={statsPageIdentifier}
-  >
-    {node}
-  </RequestContextProvider>
-);
-
 const Component = () => {
-  const {
-    platform,
-    isUK,
-    origin,
-    pageType,
-    statsDestination,
-    statsPageIdentifier,
-  } = useContext(RequestContext);
-  return (
-    <Fragment>
-      <span>{platform}</span>
-      <span>{isUK ? 'true' : 'false'}</span>
-      <span>{origin}</span>
-      <span>{pageType}</span>
-      <span>{statsDestination}</span>
-      <span>{statsPageIdentifier}</span>
-    </Fragment>
-  );
+  useContext(RequestContext);
+  return null;
 };
 
-describe('RequestContext', () => {
-  const testRequestContext = ({
-    pageType,
-    platformString,
-    statsDestination,
-    statsPageIdentifier,
-    isUK,
-    origin,
-  }) => {
-    shouldMatchSnapshot(
-      `should have a request object for platform ${platformString}, origin ${origin}, isUK, ${isUK}, pageType ${pageType}, statsDestination ${statsDestination} & statsPageIdentifier ${statsPageIdentifier}`,
-      renderWithContextProvider(<Component />, {
-        isUK,
-        origin,
-        pageType,
-        platformString,
-        statsDestination,
-        statsPageIdentifier,
-      }),
-    );
+jest.mock('react', () => {
+  const original = jest.requireActual('react');
+  return {
+    ...original,
+    useContext: jest.fn().mockImplementation(original.useContext),
   };
+});
 
-  testRequestContext({
-    pageType: 'article',
-    platformString: 'default',
-    statsDestination: 'NEWS_PS_TEST',
-    statsPageIdentifier: 'persian.articles.c0000000000o.page',
-    isUK: false,
-    origin: 'https://www.bbc.com',
-  });
-  testRequestContext({
-    pageType: 'article',
-    platformString: 'canonical',
-    statsDestination: 'NEWS_PS_TEST',
-    statsPageIdentifier: 'persian.articles.c0000000000o.page',
-    isUK: false,
-    origin: 'https://www.bbc.com',
+jest.mock('./getStatsDestination');
+jest.mock('./getStatsPageIdentifier');
+jest.mock('./getOriginContext');
+jest.mock('./getEnv');
+
+getStatsDestination.default.mockReturnValue('getStatsDestination');
+getStatsPageIdentifier.default.mockReturnValue('getStatsPageIdentifier');
+getOriginContext.default.mockReturnValue({ isUK: 'isUK', origin: 'origin' });
+getEnv.default.mockReturnValue('getEnv');
+
+const input = {
+  bbcOrigin: 'bbcOrigin',
+  id: 'id',
+  isAmp: true,
+  pageType: 'frontpage',
+  service: 'service',
+};
+
+const expectedOutput = {
+  env: 'getEnv',
+  id: input.id,
+  isUK: 'isUK',
```

Comment:
Should be same type as that expected by the Context
```suggestion
  isUK: true
```

Reply:
Done :)